cuchem/cuchem/benchmark/data.py

```diff
@@ -178,7 +178,6 @@ def fetch_n_sampling_data(self,
 
 
 class PhysChemEmbeddingData(object, metaclass=Singleton):
```

Comment:
I thought the physchem and bioactivity were going to be combined together? Up to you though.